includes/class-sensei-course-participants-widget.php

```diff
@@ -177,7 +177,7 @@ public function widget( $args, $instance ) {
 				}
 
 				if ( 'list' === $display ) {
-					$learner_name = '<h3 itemprop="name" class="learner-name">' . esc_html( $display_name ) . '</h3>' . "\n";
+					$learner_name = '<p itemprop="name" class="learner-name">' . esc_html( $display_name ) . "</p>\n";
```

Comment:
`span` would be the more semantic element choice here, as we're not displaying a paragraph of text.